### ISSUE TITLE
Solid tissue passed by the vagina 

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10119]
**Preferred term label:**
Solid tissue passed by the vagina 

**Synonyms**
Grape-like solid tissue

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Molar pregnancy

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000061
- label: Vaghinal passage of hydatidiform mole tissue
- parents: [Abnormal vaginal physiology (HP:0034268)](https://hpo.jax.org/app/browse/term/HP:0034268)
- pmid: PMID:29083593
- def: Passage from the vagina of hyatidiform tissue, appearing as clusters of vesicles (similar to small grapes) developed from the transformation of chorionic villi. This is a manifestation of a molar pregnancy (hyatidiform mole).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10119
- synonyms: Solid tissue passed by the vagina(NONE)

**Comment by kanems** [comment]
Attn @pnrobinson note the typo:
>label: Vag**h**inal passage of hydatidiform mole tissue

**Comment by pnrobinson** [comment]
@kanems 
thanks for picking up the error - fixed!
